client/app/assets/less/redash/ant.less

```diff
@@ -36,3 +42,35 @@
 .ant-dropdown-in-bootstrap-modal {
   z-index: 1050;
 }
+
+// Make Ant labels more Redash like
+.ant-form-item-label label {
+  color: inherit;
+  white-space: initial;
+  line-height: initial;
+  text-align: left;
+}
+
+// Fix ant input number showing duplicate arrows
+.ant-input-number-input::-webkit-outer-spin-button,
+.ant-input-number-input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+.ant-input-number-input {
+  -moz-appearance: textfield;
+}
+
+// Fix for text becoming black on disabled buttons
+.btn-danger, .btn-primary, .btn-success, .btn-info {
+  &[disabled] {
+    color: @white !important;
+  }
+}
+
+// Remove green border from Ant inputs
+.has-success .ant-input,
+.has-success .ant-input:hover {
+  border-color: @border-color-base !important;
+}
```

Comment:
(I'm new to Redash code so I got some n00b questions)
What does `.has-success` mean in the form? Where is it coming from?

Reply:
`.has-success` is added in [Ant's Form validation](https://ant.design/components/form/). For some reason the css for this came with a green border and I did it as an overwrite for this.

Reply:
That's weird..
According to [their less file](https://github.com/ant-design/ant-design/blob/4378454495a9c82543036c57cd10ef92ec08948f/components/form/style/index.less#L434), it should color only the success icon. Could it be our Ant package is not up to date?

Might want to research this more cause it seems the problem still persists (taken from [current netlify preview](https://deploy-preview-3209--redash-preview.netlify.com/data_sources/new):
<img width="391" alt="screen shot 2019-01-05 at 20 58 11" src="https://user-images.githubusercontent.com/486954/50728040-b2f2ca80-112c-11e9-9c02-788daa8228b4.png">

Reply:
I missed the color for `:hover` :man_facepalming: (should be blue instead). Anyway, I think the best option is to update our Ant package and see how it affects elsewhere. I remember I could find it somewhere in one of their `less` files located in `node_modules`...

Reply:
Unfortunately updating Ant was not the solution for any css problem :confused:. I'll research a bit more tomorrow to at least fix the focused and hovered border.

Reply:
I'll take a deeper look if that's ok with you.

Also, I think it is worth upgrading the package anyway in a separate PR.

Reply:
Go ahead and let me know 🙂. If I find out anything I'll mention as well.

It's of course better to find the root problem for this rather than just css it.